### PR TITLE
feat(*): update crypto

### PR DESCRIPTION
**Pull request by sandyzhou**
use seck256k1 for signature

### CODE REVIEW

File: pkg/plugins/start.go [hunk 1]
```diff
@@ -13,8 +13,8 @@ import (
 
 var logger = hclog.New(&hclog.LoggerOptions{
 	Name:   "plugin",
-	Output: os.Stdout,
-	Level:  hclog.Info,
+	Output: hclog.DefaultOutput,
+	Level:  hclog.Trace,
```
Comment (Alexader) [inline pkg/plugins/start.go]: trace level?

File: internal/txcrypto/relay_cryptor.go [hunk 2]
```diff
@@ -1,7 +1,7 @@
 package txcrypto
 
 import (
-	"crypto/elliptic"
+	"github.com/btcsuite/btcd/btcec"
```
Comment (Alexader) [inline internal/txcrypto/relay_cryptor.go:4]: import sort?

**Review by Alexader — APPROVED** [review]

File: pkg/plugins/start.go (hunk 1 above)
Comment (sandyzhou) [reply pkg/plugins/start.go]: Fixed.

File: internal/txcrypto/relay_cryptor.go (hunk 2 above)
Comment (sandyzhou) [reply internal/txcrypto/relay_cryptor.go:4]: Fixed.

**Review by jzhe886 — APPROVED** [review]

**Pull request merged**